### ISSUE TITLE
Element.createShadowRoot deprecated

### ISSUE THREAD

**Issue by jacomyma** [medialab/artoo#291]
Using the artoo.ui in Chrome triggers a warning message (see below). It seems the way to use shadow DOM has slightly changed. For your information.
```
artoo-latest.min.js:3 [Deprecation] Element.createShadowRoot is deprecated and will be removed
in M80, around February 2020. Please use Element.attachShadow instead.
See https://www.chromestatus.com/features/4507242028072960
and https://developers.google.com/web/updates/2019/07/web-components-time-to-upgrade
for more details.
```

**Comment by Yomguithereal** [comment]
Yup. I also think shadow dom was to completely eradicated from the specs and I wonder if it even still works in Firefox. Will need to rely on other tricks in the future :)

**Comment by jacomyma** [comment]
Note: since precisely today, artoo.ui crashes in Chrome. ```TypeError: this.host.createShadowRoot is not a function.```

**Comment by Yomguithereal** [comment]
It seems it was replaced by `attachShadow` and such. But it is also way more complicated now as only some specific html element can receive a shadow... Seems like fun :)

**Comment by jacomyma** [comment]
Well, as long as Artoo.ui works, I'll be a happy user 😄 
Note: if you revamp the methods, I think you can adopt a more minimalistic approach...

**Comment by Yomguithereal** [comment]
I also received some emails recently complaining about this feature not working anymore. It seems it is more used in the wild than what I thought.

**Comment by boogheta** [comment]
You guys are frightening me: does this mean google bookmarklets probably does not work anymore on some browsers?

**Comment by Yomguithereal** [comment]
Probably so. But I am on the coup :)

**Comment by Yomguithereal** [comment]
Ok, so I released a new v0.4.1 with a fix. Hopefully it should work. Can either of you @jacomyma or @boogheta tell me if this work. Normally this should not require any change from your side if you rely on the `-latest` bookmarklet of artoo (which is the default behavior). artoo declares in the console the version it is using when clicking on the bookmarklet. And maybe refresh your cache etc.

**Comment by jacomyma** [comment]
Confirmed: it re-works.
